src/components/Questions/Body.vue

```diff
@@ -70,25 +70,25 @@
         </ul>
       </div>
       <!-- subjective question answer -->
-      <!-- <div
+      <div
         v-if="isQuestionTypeSubjective"
         class="flex flex-col"
         :class="answerContainerClass"
         data-test="subjectiveAnswerContainer"
-      > -->
-      <!-- input area for the answer -->
-      <!-- <Textarea
+      >
+        <!-- input area for the answer -->
+        <Textarea
           v-model:value="subjectiveAnswer"
           class="px-2 w-full"
-          :boxStyling="subjectiveAnswerBoxStyling"
-          :placeholder="subjectiveAnswerInputPlaceholder"
-          :isDisabled="isAnswerSubmitted || previewMode"
-          :maxHeightLimit="subjectiveBoxHeightLimit"
+          boxStyling="bp-420:h-20 sm:h-28 md:h-36 px-4 placeholder-gray-400 focus:border-gray-200 focus:ring-primary"
+          placeholder="Enter your answer here"
```

Comment:
Why hardcoded strings? This won't mess with localization?

Reply:
when we add support for internationalization, the value can be added directly here.